sound/soc/sof/loader.c

```diff
@@ -192,6 +192,28 @@ static int ext_man_get_dbg_abi_info(struct snd_sof_dev *sdev,
 	return 0;
 }
 
+static int ext_man_get_config_data(struct snd_sof_dev *sdev,
+				   const struct sof_ext_man_elem_header *hdr)
+{
+	const struct sof_ext_man_config_data *config_data =
+		container_of(hdr, struct sof_ext_man_config_data, hdr);
+	int i;
+
+	for (i = 0; i < SOF_EXT_MAN_CONFIG_LAST_ELEM; i++)
+		switch (config_data->elems[i].token) {
+		case SOF_EXT_MAN_CONFIG_CAVS_LPRO:
+			sdev->clk_config = config_data->elems[i].value;
+			dev_dbg(sdev->dev, "FW clock config: %s\n", sdev->clk_config ? "LPRO" : "HPRO");
+			break;
+		default:
+			dev_dbg(sdev->dev, "unsupported token type: %d\n",
```

Comment:
Should this be dev_warn?

Reply:
@bardliao If we're going to ignore the items that the kernel cannot parse, it is better left as a dbg. warnings that are harmless are misleading

Reply:
@ranj063 I am not sure if we can ignore this or not. ``ext_man_get_config_data()`` is called when we recognize it is a SOF_EXT_MAN_ELEM_CONFIG_DATA type. Isn't that an exception when we see a SOF_EXT_MAN_ELEM_CONFIG_DATA type with an unknown token?

Reply:
@bardliao maybe for consistency with parsing the original ext_manifest header type, we can have a dev_warn here. But I have a feeling at some point this will come back to haunt us and we will end up reducing it to dev_dbg()

Reply:
@bardliao I don't think changing this to dev_warn() is good idea. Currently we have some mismatches of supporting tokens between linux driver and firmware. We support only CAVS LPRO, later we should support rest of them. Till then we will see unnecessary warnings.
And this is extended manifest and having said that ignoring unsupported tokens is good and safe approach. What do you think?

Reply:
Changing my mind. Sure, i will change this to dev_warn()